docs/conf.py

```diff
@@ -1,12 +1,7 @@
 ### path setup ####################################################################################
 
-from glob import glob
```

Comment:
I found that my local build works fine without these below definitions. Please let me know if this broke some other part of the documentation I am unaware of.